docs/meshstack.logging.md

```diff
@@ -0,0 +1,177 @@
+---
+id: meshstack.logging
+title: Logging & Auditing
+---
+
+The meshStack system consists of different components. On the one hand there are meshStack components, like the backend, replicators and further microservices. Additionally, Keycloak is used as an authentication and authorization system. Logs are therefore distributed among multiple components. In order to provide central access to the log files for administrators, a central logging system like an ELK (Elasticsearch, Logstash, Kibana) stack is used. It allows searching logs of multiple components.
+Besides log files, event tables are also used in meshStack. They can be used for auditing purposes. The events can be used to answer e.g. the following questions:
+
+- Who created a certain project?
+- Who gave or revoked a certain user access to a project?
+- Who updated the payment information on a project?
+
+The event tables always contain a reference (DB Id) to the user who created it and the timestamp of the event.
```

Comment:
DB Id... I dont think it is clear to everyone what it means

Reply:
ok, i removed it, it is not relevant here